articles/azure-sql/database/dns-alias-overview.md

```diff
@@ -17,6 +17,9 @@ ms.date: 06/26/2019
 
 Azure SQL Database has a Domain Name System (DNS) server. PowerShell and REST APIs accept [calls to create and manage DNS aliases](#anchor-powershell-code-62x) for your [logical SQL server](logical-servers.md) name.
 
+> [!IMPORTANT]
+> Currerently, for Azure Synapse Analytics, DNS alias is only supported for **Dedicated SQL Pool (Formerly DW).** For Synapse Dedicated SQL Pool, DNS alias is not supported at this stage.
+
```

Comment:
```suggestion
> [!NOTE]
> Currently, the Azure SQL Database DNS alias is only supported for Azure Synapse Analytics dedicated SQL Pool (formerly DW). For dedicated SQL pools in Synapse workspaces, the DNS alias is not currently supported.
```

Reply:
This edit has been reviewed by @ronortloff and @nanditavalsan, looks good. @xiaoyuanzi1230 see my email to get this suggestion committed and the merge conflict resolved. Thanks!